client_src/js/UI.mjs

```diff
@@ -23,13 +23,13 @@ async function show_changelog(only_if_changed) {
 		console.log("[UI] Not showing changelog.");
 		return false;
 	}
-	
+
```

Comment:
It looks like the changes in this file  are entirely whitespace related. Perhaps you could remove these changes please?